docs/analytics.md

```diff
@@ -0,0 +1,22 @@
+# Flank Analytics
+
+Flank makes use of Bugsnag as its only analytics platform. It uses Bugsnag as an error monitor and project stability tool.
+
+For more information about Bugsnag please visit its website [here](https://www.bugsnag.com/) and documentation [here](https://docs.bugsnag.com/)
+
+## Control
+
+It is possible to disable the Bugsnag analytics integration used by flank or to make use of custom Bugsnag key so that any analytics can be displayed for custom use.
+
+ - To disable, place the word ```DISABLED``` in the file ````~/.gsutil/analytics-uuid```` and Flank will no longer send any analytics data
```

Comment:
Let's mention that Flank follows the same opt out approach as GCloud CLI. That's where this file and logic comes from.

Reply:
Sure done!